boards/arm/nrf9160_pca10090/nrf9160_pca10090.yaml

```diff
@@ -9,3 +9,4 @@ ram: 64
 flash: 256
 supported:
   - i2c
+  - watchdog
```

Comment:
Will there be a problem that wdt is not enabled in DTS?

Reply:
Hmm, I haven't thought about this. I've just run the builds and they went fine.
The "status" property for the `wdt` node [is not specified](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/nordic/nrf9160_common.dtsi#L217) in _nrf9160_common.dtsi_ and the node seems to be enabled by default.

Reply:
So: does this mean that if the "status" property is missing, the default is "ok"?

Reply:
My point is that we need the device to be enabled, to get the NRF_WDT defined (at the moment) for nRF9160_pca10090/ns boards. I suppose, any WDT-related sample requires the register base address be defined...

Reply:
IF the "default" is "status ok", then I would prefer to actually add the property, so it is more visible to the reader.